bootstrap/etc/salt/cloud.maps.d/salt_masters.yml

```diff
@@ -0,0 +1,32 @@
+# -*- mode: yaml; coding: utf-8; -*-
+salt_master:
+  - master-operations-production:
+      grains:
+        roles:
+          - master
+          - master-production
+      network_interfaces:
+        - DeviceIndex: 0
+          AssociatePublicIpAddress: True
+          SubnetId: sdb://osenv/PROD_SUBNET
+          SecurityGroupId: sdb://osenv/SECURITY_GROUPS
+      tag:
+        environment: operations
```

Comment:
Do we want to make this `operations-production` for future stuff?

Reply:
We do, but that will be a separate refactoring given the current designation across a number of files.